Makefile

```diff
@@ -379,12 +379,12 @@ endif
 # The rootfs partition size is set to 512MB after 10.2.0 release (see commit 719b4d516)
 # Before 10.2.0 it was 300MB. We must maintain compatibility with older versions so rootfs size cannot exceed 300MB.
 # kubevirt and nvidia are not affected by this limitation because there no installation of kubevirt prior to 10.2.0
-# Nethertheless lets check for ROOTFS_MAXSIZE_MB not exceeding 450MB for kubevirt and ARM and 270MB for x86_64
+# Nethertheless lets check for ROOTFS_MAXSIZE_MB not exceeding 900MB for kubevirt, 450MB for ARM and 270MB for x86_64
```

Comment:
The comment is not 100% correct. On arm64, 450MB is used only for NVIDIA based platforms. I would add something like this:

```
...not exceeding 900MB for kubevirt, 450MB for NVIDIA based platforms (arm64) and 270MB for x86_64 and other arm64 platforms
```

Reply:
ok will do